server/src/test/java/org/elasticsearch/cli/TerminalTests.java

```diff
@@ -41,6 +41,8 @@ public void testVerbosity() throws Exception {
         assertPrinted(terminal, Terminal.Verbosity.VERBOSE, "text");
     }
 
+    // TODO: test error verbosity
```

Comment:
Can we add these new tests described here? I see several existing tests modified, but it would be good to have some direct coverage of the new functionality.

Reply:
Of course - thanks for catching that.